src/com_tjvendors/admin/models/fields/custom_field.php

```diff
@@ -12,6 +10,8 @@
 
 defined('JPATH_BASE') or die;
 
+use Joomla\CMS\Form\FormField;
+
 jimport('joomla.html.html');
 jimport('joomla.form.formfield');
```

Comment:
remove jimport('joomla.form.formfield'); from all field files if related class is loaded

Reply:
done